libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/TripNotificationState.kt

```diff
@@ -24,5 +24,13 @@ sealed class TripNotificationState {
     /**
      * Represents a data-less state.
      */
-    class TripNotificationFreeState internal constructor() : TripNotificationState()
+    class TripNotificationFreeState internal constructor() : TripNotificationState() {
```

Comment:
Why not make it an `object`?

Reply:
I thought about it as well, but if we wanted to add an additional property to this type in the future, it's probably better to keep it a class.